kolibri/auth/models.py

```diff
@@ -495,6 +575,18 @@ def filter_readable(self, queryset):
     def __str__(self):
         return self.get_full_name() or self.username
 
+    def has_perm(self, perm, obj=None):
+        # ensure the DeviceOwner has full access to the Django admin
+        return True
+
+    def has_perms(self, perm_list, obj=None):
+        # ensure the DeviceOwner has full access to the Django admin
+        return True
+
+    def has_module_perms(self, app_label):
+        # ensure the DeviceOwner has full access to the Django admin
+        return True
+
```

Comment:
Me, or the code? There's nothing in the *code* that depends on it, but I do find it's useful for debugging data, and we may want to be able to request local tech admins to log in there to check something or make a tweak. I don't think we should link to /admin/ within the Kolibri interface, though.

(Without this code, we have no way to access the Django admin at all, even when debugging, as we've overridden the user model -- `DeviceOwner` is the only superuser we have)